lib/modules/service.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+from data.data import Config
+
+
+class ServiceApi:
+    def __init__(self, api_url=Config.BASE_API_URL):
+        self.api_url = api_url
+
+    def service_create_entity(self, data, endpoint=Config.ENDPOINT_CREATE):
+        return requests.post(f"{self.api_url}{endpoint}", json=data)
+
+    def service_delete_entity(self, entity_id, endpoint=Config.ENDPOINT_DELETE):
+        return requests.delete(f"{self.api_url}{endpoint}{entity_id}")
+
+    def service_get_entity(self, entity_id, endpoint=Config.ENDPOINT_GET):
+        return requests.get(f"{self.api_url}{endpoint}{entity_id}")
+
+    def service_get_all_entities(self, endpoint=Config.ENDPOINT_GET_ALL):
+        return requests.get(f"{self.api_url}{endpoint}")
+
+    def service_update_entity(self, entity_id, data, endpoint=Config.ENDPOINT_PATCH):
+        return requests.patch(f"{self.api_url}{endpoint}{entity_id}", json=data)
```

Comment:
Если у нас будет не один класс с эндпоинтами, а будет несколько, которые разделены на группы, то класс Config из модуля data.py будет слишком перегружен. Лучше хранить эндпоинты в классе вместе с методами для взаимодействия с ними.
```suggestion
class ServiceApi:
    endpoint_create_url = Config.BASE_API_URL + "/api/create"
    endpoint_delete_url = Config.BASE_API_URL + "/api/delete/"
    endpoint_get_url = Config.BASE_API_URL + "/api/get/"
    endpoint_get_all_url = Config.BASE_API_URL + "/api/getAll"
    endpoint_patch_url = Config.BASE_API_URL + "/api/patch/"

    def service_create_entity(self, data):
        return requests.post(url=self.endpoint_create_url, json=data)

    def service_delete_entity(self, entity_id):
        return requests.delete(url=f"{self.endpoint_delete_url}{entity_id}")

    def service_get_entity(self, entity_id):
        return requests.get(url=self.endpoint_get_url)

    def service_get_all_entities(self):
        return requests.get(url=self.endpoint_get_all_url )

    def service_update_entity(self, entity_id, data):
        return requests.patch(url=f"{self.endpoint_patch_url}{entity_id}", json=data)
```